tmpl/head.twig

```diff
@@ -0,0 +1,11 @@
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+    <title>{% if title is defined %} {{ title }} | {% endif %}RBGreater</title>
+    {# these stylesheets and scripts are available after running "npm i" #}
+    <link rel="stylesheet" href="/node_modules/bootstrap/dist/css/bootstrap.purged.css">
+    <link rel="stylesheet" href="/node_modules/font-awesome/css/font-awesome.min.css">
+    <link rel="stylesheet" href="/node_modules/aos/dist/aos.css">
+    <link rel="stylesheet" href="/css/styles.css">
+    <script src="/node_modules/hls.js/dist/hls.min.js"></script>
```

Comment:
Any reason this script can't be at the end? This is the biggest script we have (almost 70kbyte), so loading this in the head, makes first render slow. Given we only use it on the subsites its kinda meh :)

Reply:
Yup, it's only on the watch pages now. Lighthouse performance is better as well.

Reply:
Awesome!